### PR TITLE
Theory of Computing.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Theory of Computing.js [hunk 1]
```diff
@@ -9,10 +9,10 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcv",
-	"lastUpdated": "2017-07-03 10:55:03"
+	"lastUpdated": "2019-06-10 22:52:50"
 }
 
-/*****
+/** ***
```
Comment (zuphilip) [inline Theory of Computing.js]: Let us use here simply
```js
/*
...
*/
```
and delete the other asterisks in the beginning and the end.

File: Theory of Computing.js [hunk 2]
```diff
@@ -31,10 +31,8 @@
 *****/
 
 
-
-
-/*****
-  Theory of Computing translator 
+/** ***
```
Comment (zuphilip) [inline Theory of Computing.js]: dito

File: Theory of Computing.js [hunk 3]
```diff
@@ -67,127 +65,119 @@
...
+	// We now start constructing the Zoetro item
 	var newItem = new Zotero.Item(type);
 
-	//Store the snapshot and the PDF file
-	newItem.attachments.push({
-		title: "Theory of Computing Snapshot",
-		document: doc});
+	// Store the snapshot and the PDF file
+	newItem.attachments.push({ title: "Theory of Computing Snapshot",
+		document: doc });
```
Comment (zuphilip) [inline Theory of Computing.js]: Which rule is this? This looks unfamiliar for me...

Comment (retorquere) [reply Theory of Computing.js]: There's a rule that allows such indenting, but not one that enforces it. If I put it on one line, it stays there. Alternate is to only indent the keys (which I think looks best, and which I've just submitted).

Comment (zuphilip) [reply Theory of Computing.js]: Agree, I like this option also better.

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**